packages/lune/src/builder.ts

```diff
@@ -4,40 +4,12 @@ import { createHash } from "crypto";
 import { readFile, writeFile } from "fs/promises";
```

Comment:
A more general note: the `thing.ts` + `thing/otherstuff.ts` pattern is, if functional, not idiomatic and a little odd, i'd prefer this to be e.g. `src/builder/index.ts` etc.